go.mod

```diff
@@ -173,4 +178,6 @@ replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110
 
 replace github.com/gofrs/flock => github.com/azr/flock v0.0.0-20190823144736-958d66434653
 
+replace github.com/zclconf/go-cty => github.com/azr/go-cty v1.1.1-0.20200203143058-28fcda2fe0cc
```

Comment:
The process to review my prs is taking a bit longer; so in the meantime I'll link to a git commit of my repo; hopefully everything is merged soon and we can remove this.